src/ReactiveUI.Tests/Platforms/windows-xaml/PropertyBindingTest.cs

```diff
@@ -114,10 +114,10 @@ public void TypeConvertedTwoWayBindSmokeTest()
             vm.JustADecimal = 17.2m;
             var disp1 = fixture.Bind(vm, view, x => x.JustADecimal, x => x.SomeTextBox.Text, (IObservable<Unit>?)null, null);
 
-            Assert.Equal(vm.JustADecimal.ToString(CultureInfo.InvariantCulture), view.SomeTextBox.Text);
+            Assert.Equal(vm.JustADecimal.ToString(CultureInfo.CurrentCulture), view.SomeTextBox.Text);
```

Comment:
These tests were failing on my machine, because here in Austria we have comma "," as decimal separator. I suppose this now runs on every machine, but I'll revert it if breaks something